### ISSUE TITLE
Can't scroll RNGH exported ScrollView on Android

### ISSUE THREAD

**Issue by tpcstld** [software-mansion/react-native-gesture-handler#3209]
### Description

This might be a re-regression of https://github.com/software-mansion/react-native-gesture-handler/issues/2472 not totally sure.

On an Android device with TalkBack enabled, the `ScrollView` exported from `react-native-gesture-handler` cannot be scrolled with the two-finger scroll gesture. The similar export from `react-native` does not have this problem.

There _may_ be a second-layer related issue where the scrolling _does_ work, but then after two-finger scrolling, the tap gesture (e.g. on a `Pressable`) stops working. I can't repro this personally but I thought I might mention it.

### Steps to reproduce

Here is some sample code that can repro the issue:

```typescript
import React from "react";
import {
  ScrollView as ScrollViewRN,
  Pressable,
  StyleSheet,
} from "react-native";
import {
  GestureHandlerRootView,
  ScrollView,
} from "react-native-gesture-handler";

export default function RootLayout() {
  const ScrollViewComponent = false ? ScrollView : ScrollViewRN;
  return (
    <GestureHandlerRootView
      style={[StyleSheet.absoluteFill, { backgroundColor: "red" }]}
    >
      <ScrollViewComponent>
        {[
          1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20,
          21, 22, 23, 24,
        ].map((item) => (
          <Pressable
            key={item}
            style={{
              height: 100,
              backgroundColor: `#${Math.floor(
                Math.random() * 16777215
              ).toString(16)}`,
            }}
            onPress={() => {
              console.log("pressed");
            }}
          />
        ))}
      </ScrollViewComponent>
    </GestureHandlerRootView>
  );
}
```

1. Build the app on a physical phone
2. Enable TalkBack
3. Try scrolling with two fingers up and down.

As shown in the following video, the scrolling only works if we're using the `react-native` component, and not the `react-native-gesture-handler` component.

https://github.com/user-attachments/assets/c227a018-eadd-4d9a-88ef-c0904b4cb739

### Snack or a link to a repository

https://github.com/tpcstld/rn-stuff/blob/talkback-scroll-android/app/_layout.tsx

### Gesture Handler version

2.20.2

### React Native version

0.74.5

### Platforms

Android

### JavaScript runtime

Hermes

### Workflow

None

### Architecture

Paper (Old Architecture)

### Build type

Debug mode

### Device

Real device

### Device model

Pixel 7 (Android 14)

### Acknowledgements

Yes

**Comment by ivaniuk7531** [comment]
any updates? I have a same issue

**Comment by RohovDmytro** [comment]
Confirm.

**Comment by aLemonFox** [comment]
@marcshilling This was fixed for me in 2.25.0

**Comment by marcshilling** [comment]
> [@marcshilling](https://github.com/marcshilling) This was fixed for me in 2.25.0

Confirmed! Thanks for the heads up. Guess this issue can probably be closed?

**Comment by aLemonFox** [comment]
Sorry I posted too soon after seeing my ScrollView work with MaxPointer 1 set. That does seem to be fixed.

**Comment by russriser** [comment]
This is still an issue on 2.28.0. The behavior is very inconsistent though; sometimes scrolling is smooth, other times not. The behavior differs across app opens on the same component, and also if you just toggle TalkBack off and on.

Tried a couple patches but haven't found anything that works yet.

**Comment by RohovDmytro** [comment]
By the way, I have to agree that sometimes scrolling feels amazing and other times is terrible. It got me that I though I've fixed it, but then the issue resurfaced.

Yeah. This kind of issue.

**Comment by machiapply** [comment]
Tried upgrading to 2.30.0 just now, seems to fix the issue.
-> Sorry, never mind. After upgrading Expo from 54 to 55 (RN 0.83), it stopped working again.

**Comment by bharath2468** [comment]
Hi Team, Any updates on this issue?

**Comment by m-bert** [comment]
H! Could you check if it is still reproducible on 3.0.2? I tried to look into it but it seems to work in my case.
